src/Servers/Kestrel/Core/src/Internal/Http3/QPack/HeaderField.cs

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http3.QPack
+{
+    internal readonly struct HeaderField
+    {
+        // TODO confirm this.
+        public const int RfcOverhead = 32;
+
+        public HeaderField(Span<byte> name, Span<byte> value)
+        {
+            Name = new byte[name.Length];
```

Comment:
😢

Reply:
It's only used by the static table.

Reply:
Eh nvrm it will eventually be used by the dynamic table as well.